styles.css

```diff
@@ -15,6 +30,8 @@ body{
     flex-direction: column;
```

Comment:
You are repeating the properties, you above in the `html, body` you already defined these properties

```css
    height: 100%;
    width: 100%;
```

Reply:
ok